LibreNMS/Snmptrap/Handlers/AdvaAccThresholdCrossingAlert.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * AdvaAccThresholdCrossingAlert.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Adva Threshold Exceeded Alarms.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 KanREN, Inc
+ * @author     Heath Barnhart <hbarnhart@kanren.net> & Neil Kahle <nkahle@kanren.net>
+ */
+
+namespace LibreNMS\Snmptrap\Handlers;
+
+use App\Models\Device;
+use LibreNMS\Interfaces\SnmptrapHandler;
+use LibreNMS\Snmptrap\Trap;
+
+class AdvaAccThresholdCrossingAlert implements SnmptrapHandler
+{
+    /**
+     * Handle snmptrap.
+     * Data is pre-parsed and delivered as a Trap.
+     *
+     * @param Device $device
+     * @param Trap $trap
+     * @return void
+     */
+    public function handle(Device $device, Trap $trap)
+    {
+        $raw = $trap->getRaw();
+        Log::event($raw, $device->device_id, 'trap', 2);
```

Comment:
Dumping the entire trap into the eventlog in addition to the event below might be too much noise. (could this be debug code?)

Reply:
I'm was wanting to submit this now under the idea of something is better than nothing, then circle back and improve it later. If you prefer I can remove it from this pull request and add it at a later time. I'd like to get the other traps included as soon as possible as coding out all cases for this trap might take some time.

This is an interface threshold trap that fires on a number user defined thresholds. For example it will send a trap if x number of 64 byte packets are received on an interface or if the interface is up but not passing any traffic. There are 48 thresholds that are monitored at 15 minute and 1 day intervals. So this particular handler has 96 possible cases. Additionally this is only for "access port" interfaces, there is a similar handler for "network port" interfaces. Hopefully I can do one and do a find and replace on the OID names, but it will take a bit to work.

How would you like to proceed?

Reply:
Ok, I should probably have my coffee, read my code, then respond. Yep, there was debugging still in there. Going to fix that.

Reply:
Fixed the debug and added the requested functionality.